File: LocationService/src/main/java/com/stackroute/educationservice/springlocation/domain/PastLocation.java
package com.stackroute.educationservice.springlocation.domain;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class PastLocation {
    private String pastLocationId;
    private String cityName;
    private String stateName;
    private String pincode;
}

File: Angular/src/app/add-module/location-dialog/domain/pastlocation.ts
export class PastLocation {
    constructor(private pastLocationId: string,
                private cityName: string,
                private stateName: string,
                private pincode: string) {
                    this.pastLocationId = pastLocationId;
                    this.cityName = cityName;
                    this.stateName = stateName;
                    this.pincode = pincode;
                }
}
